executor/executor.go

```diff
@@ -1263,7 +1263,7 @@ func init() {
 			ctx = opentracing.ContextWithSpan(ctx, span1)
 		}
 
-		e := &executorBuilder{is: is, ctx: sctx}
+		e := newExecutorBuilder(sctx, is, nil, "")
```

Comment:
Passing empty string is a bit consufing.

Reply:
This is to make the modified code equals the previous. However passing scope to executorBuilder is not a right way, the txnScope should also be read from context, it should be refactored later too.